### PR TITLE
Tutorials image upload

### PR DESCRIPTION
**Pull request by jh-RLI**
This PR adds the functionality to upload images to the OEP (media root). This is used in the tutorials application. A user can upload images via drag-n-drop, C&P or file context menu while creating or editing a tutorial.

## Known Issues
- **If a user uploads images in a large size, the tutorial detail view will render a huge image.**
- The button for uploading images got the icon symbol as the "Insert markdown syntax for links" button.

### CODE REVIEW

**Comment by MGlauer** [comment]
The storage path of the pictures should incorporate the `MEDIA_ROOT` setting from the security settings in some way as this is where the media is stored.

**Comment by jh-RLI** [comment]
I used the "ImageField" and "upload_to=" which uses the MEDIA_ROOT by default. Seehttps://docs.djangoproject.com/en/3.1/topics/files/

**Comment by jh-RLI** [comment]
I used the "ImageField" and "upload_to=" which uses the MEDIA_ROOT by default. See https://docs.djangoproject.com/en/3.1/topics/files/ .

File: tutorials/templates/add.html [hunk 1]
```diff
@@ -36,7 +36,14 @@ <h2 class="site-header">
...
     <script src="https://unpkg.com/easymde/dist/easymde.min.js"></script>
     <script>
-        var easyMDE = new EasyMDE({element: document.getElementById('id_markdown'), nativeSpellcheck: false, forceSync: true});
+        var easyMDE = new EasyMDE({element: document.getElementById('id_markdown'),
+                                    nativeSpellcheck: false,
+                                    forceSync: true,
+                                    showIcons: ['upload-image'],
+                                    uploadImage: true,
+                                    imageMaxSize: '100px',
+                                    imageUploadEndpoint: 'http://openenergy-platform.org/api/v0/image/upload/',
```
Comment (MGlauer) [inline tutorials/templates/add.html:45]: The URL should not be hard-coded
```suggestion
                                    imageUploadEndpoint: '/api/v0/image/upload/',
```

**Review by MGlauer — CHANGES_REQUESTED** [review]

**Comment by jh-RLI** [comment]
What was the problem here? @MGlauer

**Comment by MGlauer** [comment]
Sorry, thought I merged it. Patched it.

**Comment by jh-RLI** [comment]
Thanks :)

**Pull request closed without merging**